package.json

```diff
@@ -28,6 +44,9 @@
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
+    "lint:types": "glint",
+    "prepack": "glint -d",
```

Comment:
This means we technically will already be publishing types once we add a .ts or .gts file, but I don't think that's a huge issue unless we document how to use them in apps. We consider these private until the docs are updated.

This makes testing the types easier and adventurous consumers can start using them (with the knowledge that things might break between releases).